### ISSUE TITLE
Base Doc - Data Model Suggestion for AchievementCredential (credentialStatus)

### ISSUE THREAD

**Issue by kayaelle** [IMSGlobal/openbadges-specification#414]
Suggest pointing to vc-model directly unless/until Open Badges makes recommendations for how their implementers should do this.

**Comment by amiller-ims** [comment]
In line with the Charter, I wrote a [Verification section](https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#verification) for OB 3.0 which is the OB 2.0 Verification process updated for OB 3.0 and using VC 1.1 Status, Refreshing, and Schema extensions. I also wrote the corresponding [VC Extensions](https://imsglobal.github.io/openbadges-specification/ob_v3p0.html#vc-extensions) to replicate status, refreshing, and schema checking from OB 2.0.

Of course the project group can decide to change verification in OB 3.0, but this approach (replicating the OB 2.0 verification process) is simple and I'm not aware of any complaints about the OB 2.0 process.
